tools/cli/internal/cli/split/split_test.go

```diff
@@ -78,6 +78,26 @@ func TestSplitPrivatePreviewRun(t *testing.T) {
 	require.Contains(t, info.Spec.Paths.Map(), "/api/atlas/v2/groups")
 }
 
+func TestSplitUpcomingRun(t *testing.T) {
+	t.Parallel()
+	fs := afero.NewMemMapFs()
+	opts := &Opts{
+		basePath:   "../../../test/data/openapi_with_upcoming.json",
+		outputPath: "foas.yaml",
+		fs:         fs,
+		env:        "dev",
```

Comment:
the-absolute-super-nit: set format to yaml  so that you only generate a yaml as "all" is the default and we'll generate an unused file

Reply:
I will update it

Reply:
sorry i approved can be in your next PR